activesupport/test/cache/behaviors/cache_store_behavior.rb

```diff
@@ -174,6 +174,16 @@ def test_write_multi_empty_hash
     assert @cache.write_multi({})
   end
 
+  def test_write_multi_expires_in
+    time = Time.now
+    key = SecureRandom.uuid
+    @cache.write_multi({ "#{key}" => 1 }, expires_in: 10)
+
+    Time.stub(:now, time + 11) do
```

Comment:
Is there a reason to not use [`travel`](https://edgeapi.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel) here?

Reply:
Yeah. Copy paste from the above test.
Just noticed another problem - `"#{key}"` , interpolation is not needed. A work for another rubocop cop `Style/RedundantInterpolation`. I noticed a few offenses of it in rails. Do you think its worth enabling?

Reply:
Personally, no.  Although I can't remember specific examples, I know there have been times when I preferred to write `"#{foo}"` instead of `foo.to_s` for stylistic reasons.

Also, given Ruby's lack of static typing, it seems like such a cop might not be very reliable.

Reply:
Yeah, no problems. This cop just detects patterns like `"#{foo}"` and suggests to replace them with `foo.to_s` (as a safe option), but many times this can be just written as `foo`.